ols/constants.py

```diff
@@ -163,7 +163,15 @@ class GenericLLMParameters:
 
 # Supported attachment types
 ATTACHMENT_TYPES = frozenset(
-    {"log", "configuration", "error message", "stack trace", "api object"}
+    {
+        "log",
+        "configuration",
+        "error message",
+        "stack trace",
+        "api object",
+        "event",
+        "alert",
```

Comment:
please sort these alpabetically, it will make it easier in the future when people are looking to see if a particular attachment type exists or add a new one.

Reply:
sorted @bparees